packages/driver/src/cypress/stack_utils.js

```diff
@@ -288,7 +296,7 @@ const normalizedUserInvocationStack = (userInvocationStack) => {
   // whereas Chromium browsers have the user's line first
   const stackLines = getStackLines(userInvocationStack)
   const winnowedStackLines = _.reject(stackLines, (line) => {
-    return line.includes('cy[name]') || line.includes('Chainer.prototype[key]')
+    return line.includes('cypress:///')
```

Comment:
I'm not sure about this change. Keep in mind `cypress:///` only appears in development and not production, so while the tests may pass for this, the effect will be different in production.

Reply:
ok, well then this wont work...